{{cookiecutter.project_slug}}/.github/workflows/main.yml

```diff
@@ -0,0 +1,47 @@
+{% raw %}
+name: build ⚙️
+
+on: [ push, pull_request ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    steps:
+    - name: Checkout repository and submodules
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - name: Install packages
+      run: |
+        sudo apt-get -y install pandoc
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install requirements 📦
+      run: |
+        python -m pip install --upgrade pip
+        pip install -e .
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        if [ -f requirements_dev.txt ]; then pip install -r requirements_dev.txt; fi
+    - name: Test with pytest ⚙️
+      run: make test
+    - name: Lint with flake8 ⚙️
+      run: make lint
+      if: matrix.python-version == 3.6
+    # - name: Check formatting with black ⚙️
+    #   run: black --check --target-version py36 birdy tests
```

Comment:
Given that this is `raw`, I'm guessing we can't easily replace the project name with jinja2 templating, i.e. `{{ cookiecutter.project_slug }}? Might be cleaner to remove this commented-out call.